torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -319,7 +319,7 @@ def codegen_inputs(self):
                 )
                 self.prefix.splice(
                     f"""
-                    if ((long({input_name}.data_ptr()) & ({GPU_ALIGN_BYTES} -1)) != 0) {{
+                    if ((reinterpret_cast<std::uintptr_t>({input_name}.data_ptr()) & ({GPU_ALIGN_BYTES} -1)) != 0) {{
```

Comment:
has to change this for windows cross-compilation. This should also work for linux.